### PR TITLE
Server

### PR DESCRIPTION
**Pull request by GovindSuresh**
Adds the cloud-config file

### CODE REVIEW

File: cloud-config.txt [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#cloud-config
+
+#v1.0
+#Created two user accounts for the server admins. Automatically adds public keys for connection and provides passwordless
+#sudo access. The configuration also edits the sshd_config file to prevent root ssh access and allows ssh access from the two # users
```
Comment (rdkr) [inline cloud-config.txt]: can you update these docs and move them to the README, along with some usage instructions pls?

**Review by rdkr — CHANGES_REQUESTED** [review]

File: cloud-config.txt (hunk 1 above)
Comment (GovindSuresh) [reply cloud-config.txt]: Done friend

File: README.md [hunk 2]
```diff
@@ -6,8 +6,20 @@ Project Code is a collaborative project to learn and teach Python, DevOps, and a
 * project management - see [trello](https://trello.com/b/FC8mke6j/project-code)
 * members - see [CODEOWNERS](CODEOWNERS) and [AUTHORS](AUTHORS)
 
+
 ## Hosting
+### Cloud-init.txt 
+This is the cloud-init file that configures the server, it creates the main user accounts for the code-owners and provides passwordless sudo access via SSH. The file also edits the sshd_config file to prevent root ssh access.
+As we are using DigitalOcean, to use the cloud-init simply select the user-data box in the 'select additional options' section when creating a droplet and copy in the contents of th
```
Comment (rdkr) [inline README.md]: wat?

File: README.md [hunk 3]
```diff
@@ -6,8 +6,20 @@ Project Code is a collaborative project to learn and teach Python, DevOps, and a
...
 ## Application
 
 ## CI/CD
+
+
+
+
+
+
+
```
Comment (rdkr) [inline README.md]: pls trim to one blank line at end of file

File: README.md [hunk 4]
```diff
@@ -6,8 +6,20 @@ Project Code is a collaborative project to learn and teach Python, DevOps, and a
 * project management - see [trello](https://trello.com/b/FC8mke6j/project-code)
 * members - see [CODEOWNERS](CODEOWNERS) and [AUTHORS](AUTHORS)
 
+
 ## Hosting
+### Cloud-init.txt 
+This is the cloud-init file that configures the server, it creates the main user accounts for the code-owners and provides passwordless sudo access via SSH. The file also edits the sshd_config file to prevent root ssh access.
```
Comment (rdkr) [inline README.md]: grammar

File: cloud-config.txt [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        https.list:
+            source: "deb http://security.ubuntu.com/ubuntu xenial-security main"
+            
+        docker.list:
+            source: "deb [arch=amd64] https://download.docker.com/linux/ubuntu xenial stable"
+            keyid: 0EBFCD88 #docker gpg key
+            keyserver: hkp://keyserver.ubuntu.com:80
+
+#installs packages from the repos defined above             
+runcmd:
```
Comment (rdkr) [inline cloud-config.txt]: `runcmd` is needed as well as `packages`?

**Review by rdkr — CHANGES_REQUESTED** [review]
nice. pls check if `packages` and `runcmd` are trying to do same thing?

needs some tidying. also can we have a space between `#` and the comment for readability

**Review by jcraggs — APPROVED** [review]

**Review by rdkr — APPROVED** [review]

**Pull request merged**